doc/faq.rst

```diff
@@ -330,3 +330,21 @@ However, a global random state is prone to modification by other code during
 execution. Thus, the only way to ensure replicability is to pass ``RandomState``
 instances everywhere and ensure that both estimators and cross-validation
 splitters have their ``random_state`` parameter set.
+
+Why do categorical variables need preprocessing in scikit-learn, compared to other tools?
+--------------------------------------------------------------------------------
+
+Most of scikit-learn assumes data is in NumPy arrays or SciPy sparse matrices of
+a single numeric dtype. These do not directly explicitly categorical variables at
```

Comment:
Typo: the verb is missing ("These do not directly explicitly [represent] categorical variables")

Reply:
Thanks